app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,27 @@
+.alert-error {
+    background-color: #f2dede;
+    border-color: #eed3d7;
```

Comment:
Color literals like `#eed3d7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces